src/components/Layout/DrawerAppBar/DrawerAppBar.js

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import MenuIcon from '@mui/icons-material/Menu';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import {Outlet, useNavigate} from "react-router-dom";
+import { styled, alpha } from '@mui/material/styles';
+import SearchIcon from '@mui/icons-material/Search';
+import {InputBase} from "@mui/material";
+
+const drawerWidth = 240;
+const navItems = [
+    {name: 'Home', path: ''},
```

Comment:
you don't need the Home route, you can add an `onCLick()` handler on the `Book App` title that navigates to `/`